src/stability_sdk/animation.py

```diff
@@ -781,6 +753,21 @@ def render_stereo_eye_views(self, frame_idx, frame: np.ndarray) -> np.ndarray:
         
         return np.concatenate(eye_images, axis=1)
 
+    def save_settings(self, filename: str):
+        settings_filepath = os.path.join(self.out_dir, filename) if self.out_dir else filename
+        with open(settings_filepath, "w+", encoding="utf-8") as f:
```

Comment:
I think "w" could be used here since I don't see reading? It's unimportant, but may be slightly nicer to use "w" if it works.

Reply:
Good eye! Yeah that just needs to be "w". Not sure why that was ever "w+" 🤔